src/components/Onboarding/derivationPath/DerivationPath.tsx

```diff
@@ -59,20 +100,55 @@ export const DerivationPath = ({
       title="Derivation Path"
     >
       <form onSubmit={handleSubmit(onSubmit)} style={{ width: "100%" }}>
-        <FormControl marginBottom="20px">
-          <FormLabel>Select Path</FormLabel>
-          <Select
-            onChange={newVal => setValue("derivationPath", newVal)}
-            options={AVAILABLE_DERIVATION_PATHS}
-            selected={DEFAULT_DERIVATION_PATH}
+        <Center width="100%" marginBottom="12px">
+          <Text fontWeight={isCustomPath ? 400 : 600} size="sm">
+            Default Path
+          </Text>
+          <Switch
+            data-testid="custom-path-switch"
+            marginX="10px"
+            onChange={() => setIsCustomPath(v => !v)}
+            variant="danger"
           />
-          {errors.derivationPath && (
-            <FormErrorMessage data-testid="error-message">
-              {errors.derivationPath.message}
-            </FormErrorMessage>
-          )}
-        </FormControl>
-        <Button width="100%" marginTop="12px" size="lg" type="submit">
+          <Text fontWeight={isCustomPath ? 600 : 400} size="sm">
+            Custom Path
+          </Text>
+        </Center>
+        {isCustomPath && (
+          <>
+            <Center marginTop="16px" marginBottom="32px">
+              <Text width="340px" color={colors.orange} textAlign="center" size="xs">
+                Please write down to your derivation path. You may not be able to restore your data
```

Comment:
Typo?

"Please write down your derivation path. You may not be able to restore your data without it"